tests/kubernetes/kind/upgrade.yml

```diff
@@ -0,0 +1,12 @@
+# Upgrade the MongoDB server version from 5.0.8 to 5.0.12. That will cause a rolling deploy where
```

Comment:
Where is the 5.0.8 version specified?

Reply:
5.0.8 is specified in the `MongoDBCommunity` resource in [mongo.yml](https://github.com/mongodb-labs/drivers-atlas-testing/blob/8b3f72b040045399ed4cd9e35d440f8eea314bf1/kubernetes/kind/mongodb.yml#L21).

Reply:
Good idea, will add comments!

Reply:
Added comments in https://github.com/mongodb-labs/drivers-atlas-testing/pull/151.